sql/mozfun/norm/vpn_attribution/udf.sql

```diff
@@ -66,100 +67,58 @@ RETURNS STRUCT<
         CASE
         WHEN
-          utm_source = "firefox-browser"
+          utm.source = "firefox-browser"
         THEN
           "Website: Firefox Browser"
         WHEN
-          utm_medium = '["referral","referral"]'
+          utm.medium = '["referral","referral"]'
           -- preserve legacy behavior
           AND (referrer LIKE "%app-store%") IS NOT TRUE
         THEN
           "Possible Attribution Error - Monitor"
         WHEN
           referrer LIKE "%app-store%"
-          AND utm_medium IS NULL
-          AND utm_source IS NULL
-          AND utm_campaign IS NULL
-          AND utm_content IS NULL
+          AND utm = ("(not set)", "(not set)", "(none)", "(direct)")
         THEN
           CONCAT("App Store: ", SPLIT(referrer, "-")[offset(2)])
         WHEN
           provider = "Apple Store IAP"
         THEN
           provider
         WHEN
-          normalized_medium IS NOT NULL
-          OR utm_source IS NOT NULL
-          OR utm_campaign IS NOT NULL
-          OR utm_content IS NOT NULL
+          normalized_medium != "(none)"
+          OR utm != ("(not set)", "(not set)", "(none)", "(direct)")
+          OR referrer IS NULL
         THEN
           "Website"
         ELSE
           "Unknown"
         END
         AS normalized_acquisition_channel,
-        -- normalized_campaign
-        IF(
-          utm_campaign IS NULL
-          AND (
-            (utm_content IS NULL AND utm_medium IS NULL AND utm_source IS NULL AND referrer IS NULL)
-            OR utm_source = "firefox-browser"
-            OR NULLIF(normalized_medium, '["referral","referral"]') IS NOT NULL
-          ),
-          "(not set)",
-          utm_campaign
-        ) AS normalized_campaign,
-        -- normalized_content
-        IF(
-          utm_content IS NULL
-          AND (
-            (
-              utm_campaign IS NULL
-              AND utm_medium IS NULL
-              AND utm_source IS NULL
-              AND referrer IS NULL
-            )
-            OR utm_source = "firefox-browser"
-            OR NULLIF(normalized_medium, '["referral","referral"]') IS NOT NULL
-          ),
-          "(not set)",
-          utm_content
-        ) AS normalized_content,
-        --
+        utm.campaign AS normalized_campaign,
+        utm.content AS normalized_content,
         normalized_medium,
         -- normalized_source
         CASE
         WHEN
-          utm_medium IS NULL
-          AND utm_source IS NULL
-          AND utm_content IS NULL
-          AND utm_campaign IS NULL
+          utm.source LIKE "mozilla.org-whatsnew%"
         THEN
-          CASE
-          WHEN
-            organic_referrer IS NOT NULL
-          THEN
-            organic_referrer
-          WHEN
-            internal_referrer IS NOT NULL
-          THEN
-            internal_referrer
-          WHEN
-            referrer IS NULL
-          THEN
-            "(direct)"
-          END
+          "www.mozilla.org-whatsnew"
```

Comment:
`utm_source` was previously formatted like `mozilla.org-whatsnew99`. the new format in the guardian DB is `www.mozilla.org-whatsnew` with no version suffix. `utm_campaign` continues to be formatted like `whatsnew99`. Normalizing the old value to the new format will require back filling `all_subscriptions_v1`.